pavement.py

```diff
@@ -210,12 +231,27 @@ def test_ci(args):
     """
     Run tests in continuous integration environment.
     """
+    default_args = [
+        '--with-run-reporter',
+        '--with-timer',
+        ]
+
+    # Show some info about the current environment.
+    from OpenSSL import SSL, __version__ as pyopenssl_version
+    from coverage.cmdline import main as coverage_main
+
+    print 'PYTHON %s' % (sys.version,)
```

Comment:
yes... we need to update pocket-lint to work with pylint so that we include all the files in py3 tests.

but this is for another ticket